thoth/user_api/api_v1.py

```diff
@@ -80,9 +81,46 @@
 
 _PROVENANCE_CHECK_PROTECTED_FIELDS = frozenset({"kebechet_metadata"})
 
+# custom metric to expose cache hit rates for different service endpoints
+def _create_metrics_service_cache_hits(services: typing.List[str]):
+    service_request_cache_hit_rate = {}
+
+    for service in services:
+        service_request_cache_hit_rate[service] = {}
+        service_request_cache_hit_rate[service]["authenticated"] = metrics.counter(
+            "thoth_cache_hit_rate",
```

Comment:
should we be a little bit more specific on this? user_api_cache_hit ?!

Reply:
Updated in https://github.com/thoth-station/user-api/pull/1350/commits/cbdf2841c03851f93caf3fb7d85f8b52c1c49646